server/pa/topics_test.py

```diff
@@ -0,0 +1,11 @@
+
+import unittest
+from .topics import get_topics
+
+
+class TopicTestCase(unittest.TestCase):
+
+    def test_get_topics(self):
+        topics = get_topics()
+        self.assertEqual(321, len(topics))
```

Comment:
We need to change the values every time we add/remove a topic. Can it be generic here?